src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+unreleased
+^^^^^^^^^^^^^^^^^^
+
+* Include Datalake Analytics and Datalake Store modules.
```

Comment:
Minor: Data Lake is two words so:
Include Data Lake Analytics and Data Lake Store modules